resources/js/Pages/Error.vue

```diff
@@ -0,0 +1,35 @@
+<script setup>
+import { ArrowLeftIcon } from '@heroicons/vue/24/outline'
+
+defineProps({
+  statusCode: Number,
+})
+</script>
+
+<template>
+  <div class="grid min-h-screen grid-cols-1 grid-rows-[1fr,auto,1fr] bg-white lg:grid-cols-[max(50%,36rem),1fr]">
+    <main class="mx-auto w-full max-w-7xl px-6 py-24 sm:py-32 lg:col-span-2 lg:col-start-1 lg:row-start-2 lg:px-8">
+      <div class="max-w-lg">
+        <p class="text-2xl font-semibold leading-8 text-blumilk-500">
+          {{ statusCode }}
+        </p>
+        <h1 class="mt-4 text-4xl font-bold tracking-tight text-gray-900 sm:text-7xl">
+          Page not found
```

Comment:
Title and description is always the same, even if the user gets a non-404 error.

![image](https://github.com/blumilksoftware/escooters/assets/56546832/80855548-de22-47d3-a042-d4dcb329afbf)

 It shouldn't be like that.
 
 
 Also I think that when the user gets 403 error, we should show 404 error. Returning a 404 HTTP error instead of a 403 can enhance security by not disclosing the existence of a resource, provide a better user experience by reducing confusion, and aid compliance with privacy laws in certain jurisdictions. The choice between a 403 and a 404 depends on the context and specific use case.